script/release/release.js

```diff
@@ -103,6 +103,7 @@ function assetsForVersion (version, validatingRelease) {
     `electron-${version}-linux-armv7l.zip`,
     `electron-${version}-linux-ia32-symbols.zip`,
     `electron-${version}-linux-ia32.zip`,
+    `electron-${version}-linux-x64-debug.zip`,
```

Comment:
Need to add all linux arches here:
```suggestion
    `electron-${version}-linux-x64-debug.zip`,
    `electron-${version}-linux-ia32-debug.zip`,
    `electron-${version}-linux-linux-armv7l-debug.zip`,
    `electron-${version}-linux-arm64-debug.zip`,
```

Reply:
It is only available for x64, see above comment that I re-opened and I CCd you

Reply:
```
@nornagon on Jul 2 
given this script should never be invoked on win or mac, and given we don't support 32-bit linux any more (https://electronjs.org/blog/linux-32bit-support), just assert that we're on linux64 in main().
```

Reply:
@CapOM if the intention is only to produce these for linux x64, then we shouldn't run the scripts that create the debug dirs/zips on the other arches.

Reply:
Also, we fully support arm/arm64 so, I'm not sure why we wouldn't generate those files there unless there is a technical reason that we can't support arm/arm64

Reply:
I added the arm and arm64 zips here

Reply:
Nice!

Reply:
@nornagon actually are you sure it works ? See `get_objcopy_path` implementation, it skips if it is not x64.

Reply:
uh, yeah, you're right. I'll revert this for now and we can at least get this working on x64.